### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by russellb**
Adopt the Contributor Covenant code of conduct for Nexodus.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+contact@nexodus.io.
```
[the remark this answers is not in the document]
Comment (russellb) [reply CODE_OF_CONDUCT.md:63]: No, it's just me.

An alternative would be to list individual email addresses.

Or we could link to https://github.com/orgs/nexodus-io/people (and have everyone update their membership to public).

Or create a `MAINTAINERS.md` file with maintainer contact info and point to that ...

[the remark this answers is not in the document]
Comment (dave-tucker) [reply CODE_OF_CONDUCT.md:63]: Or make contact@nexodus.io a mailing list. I don't mind any mechanism so long as it's actively monitored by at least 2 community leaders.

[the remark this answers is not in the document]
Comment (russellb) [reply CODE_OF_CONDUCT.md:63]: Yeah, good point.

The email address is just namecheap email forwarding. I don't see anywhere that says it supports forwarding to multiple recipients. I'll come up with something.

[the remark this answers is not in the document]
Comment (russellb) [reply CODE_OF_CONDUCT.md:63]: done, added a maintainers file that is referenced

**Review by vishnoianil — APPROVED** [review]

**Review by nerdalert — APPROVED** [review]

**Pull request merged**